scaffolder-templates/basic-workflow/template.yaml

```diff
@@ -141,10 +117,62 @@ spec:
                   description: Private SSH key associated with the GitHub account - Use Base64 encoding
                   ui:widget: password
                   ui:options: 'rows: 10'
+                persistenceEnabled:
+                  title: Enable Persistance
+                  type: boolean
+                  description: Indicates whether the workflow shall have persistence enabled.
+                  default: false
               required:
                 - namespace
                 - argocdNamespace
                 - sshPrivateKey
+                - persistenceEnabled
+              dependencies:
+                persistenceEnabled:
+                  oneOf:
+                      - properties:
+                          persistenceEnabled:
+                            const: false
+                      - properties:
+                          persistenceEnabled:
+                            const: true
+                          persistencePSQLSecretName:
```

Comment:
somewhere (here, the chart, the website?) the user needs some instructions on how to build this secret

Reply:
a blog post will explain how to configure the postgreSQL I guess, but right now, using the same as what is described in the postgreSQL chart in the orchestrator

But this is only temporary changes, this will be reverted once the operator support the feature via the sonataflowplatform